src/app/index.ts

```diff
@@ -5,7 +5,9 @@
  * of the MIT license.  See the LICENSE.txt file for details.
  */
 
-import "zone.js";
+//import "zone.js";
```

Comment:
We should add a comment here explaining the reason for this (explicitly including part of zone.js instead of the whole thing).